### ISSUE TITLE
Suggest me an operation to add.

### ISSUE THREAD

**Issue by cmvyas** [cmvyas/App_Guide#7]

**Comment by iaryangoyal** [comment]
@cmvyas I have an Option

**Comment by cmvyas** [comment]
@iaryangoyal  do tell me

**Comment by iaryangoyal** [comment]
Use of Google-Pay can be elaborate

**Comment by nitin-pandita** [comment]
You can add  

Use Google pay to link you bank account

**Comment by Ankur122701** [comment]
How to buy a product from Amazon, Flipkart.

**Comment by cmvyas** [comment]
@its-harsshhh go for it

**Comment by its-harsshhh** [comment]
@we can also add regional languages to it so that its reach to a larger audience!

**Comment by PreetiBaskey** [comment]
Yeah.. we can't do it easily for all languages but may be with some, we need to list out them first i think

**Comment by its-harsshhh** [comment]
@cmvyas please take a look on my recent-pull request!

**Comment by PreetiBaskey** [comment]
@cmvyas ,  @its-harsshhh   ok and one more think, in basic common payment apps like PhonePay... there is continuous sliding of new deals or something like that.. we can avoid that in our case...

**Comment by its-harsshhh** [comment]
@PreetiBaskey literally your designs are awesome! Its blew my mind

**Comment by its-harsshhh** [comment]
can i also join you @PreetiBaskey  in this, i have a little bit interest in UI designs
what you say?
